docs/content/preview/yugabyte-platform/create-deployments/connect-to-universe.md

```diff
@@ -21,13 +21,36 @@ You can connect to the database on a universe in the following ways:
 
 ## Download the universe certificate
 
-If the universe uses encryption in transit, to connect you need to first download the universe TLS root certificate. Do the following:
+If the universe uses Client-to-Node encryption in transit, to connect you need to first download the universe TLS certificate. Do the following:
 
 1. Navigate to **Configs > Security > Encryption in Transit**.
 
-1. Find the certificate for your universe in the list and click **Actions** and download the certificate.
+1. Find your universe in the list.
 
-For more information on connecting to TLS-enabled universes, refer to [Connect to clusters](../../security/enable-encryption-in-transit/#connect-to-clusters).
+1. Download the certificate.
+
+    - If you are connecting using a YSQL client (such as ysqlsh), click **Actions**, and choose **Download YSQL Cert**.
+
+        This downloads the `yugabytedb.crt` and `yugabytedb.key` files.
+
+    - If you are connecting using a YCQL client (such as ycqlsh), click **Actions**, and choose **Download Root CA Cert**.
+
+        This downloads the `root.crt` file.
+
+    - If you are connecting to universes that are configured with custom CA-signed certificates, obtain the root CA and client YSQL certificate from your administrator. These certificates are not available on YugabyteDB Anywhere for downloading.
```

Comment:
```suggestion
```

Technically, the root CA is also available from YBA because they have uploaded it to us. The client YSQL cert is only available on their side but it is rarely used and like I commented above, I think we can ignore it in docs.